packages/dev/docs/pages/react-aria/accessibility.mdx

```diff
@@ -104,3 +104,12 @@ All React Aria components are tested across a wide variety of devices, browsers,
 * [NVDA](https://www.nvaccess.org) on Windows in Firefox and Chrome
 * [VoiceOver on iOS](https://www.apple.com/accessibility/iphone/vision/)
 * [TalkBack](https://www.android.com/accessibility/) on Android in Chrome
+
+### False positives
+
+There are a number of known accessibility false positives in React Aria and React Spectrum, currently documented here in our [wiki](https://github.com/adobe/react-spectrum/wiki/Known-accessibility-false-positives).
```

Comment:
IMO we should continue to maintain this wiki as our source of truth for these accessibility data attributes rather than maintain a table of them in this page here, open to opinions though